src/Core/Version.php

```diff
@@ -8,7 +8,7 @@
  */
 final class Version
 {
-    public const VERSION = '8.1.0';
+    public const VERSION = '9.0.0';
```

Comment:
You also need to update the const below `MAJOR_VERSION` is 9 `MINOR_VERSION` is 0

Reply:
Done